src/md/2011-12-16-negative-zero-equal-but-not-equal.md

```diff
@@ -1,6 +1,8 @@
 ```
-    0 === -0        //true
-    1/0 === 1/-0    //false
+   +0 ==   0     //true
+   -0 ==  +0     //true
```

Comment:
Those two should probably be `===` (the result is still the same)

Reply:
just for pure understading :)
and my it's be doubt for beginner understading between 0,-0